homeassistant/components/apple_tv/media_player.py

```diff
@@ -242,12 +253,13 @@ def _is_feature_available(self, feature):
 
     async def async_turn_on(self):
         """Turn the media player on."""
-        await self.manager.connect()
+        if self.atv:
+            await self.atv.power.turn_on()
 
     async def async_turn_off(self):
         """Turn the media player off."""
-        self._playing = None
-        await self.manager.disconnect()
+        if self.atv and self.atv.power.power_state == PowerState.On:
```

Comment:
Removed the previous login about disconnecting from device. this will keep listening to the device when turning it on without HA.
Also I don't check if turn on/off feature is available, since I prefer to get an error, so the user will get the error.